src/deploy/functions/prepareFunctionsUpload.ts

```diff
@@ -102,13 +112,14 @@ async function packageSource(
       filesize(archive.pointer()) +
       ") for uploading"
   );
-  return tmpFile;
+  const hash = fileHashes.join(".");
+  return { pathToSource: tmpFile, hash: hash };
```

Comment:
Couldn't we shorten this to `return { pathToSource: tmpFile, hash };`

Reply:
Good catch!